### PR TITLE
Step 10

### PR DESCRIPTION
**Pull request by ahunter1**
Added command line for Windows users in step 10

### CODE REVIEW

**Comment by talumbau** [comment]
yep, good catch. thanks!

File: docs/source/contributor_guide.rst [hunk 1]
```diff
@@ -80,6 +80,12 @@ Setup Git
 
        tax-calculator$ source activate taxcalc-dev
 
+If you are working on Windows, use the following the command line:
```
Comment (MattHJensen) [inline docs/source/contributor_guide.rst]: Is this missing a "from"? "from the command line"? 

Also, when I view the file [here](https://github.com/ahunter1/Tax-Calculator/blob/patch-1/docs/source/contributor_guide.rst), I see that the formatting is a bit off. Try indenting the line starting "if you are working on windows..." and see if that fixes it.

You will do all this by just adding another commit or two locally to the same branch and then pushing to your fork. The PR will update automatically.

**Comment by MattHJensen** [comment]
@talumbau said

> yep, good catch. thanks!

Agreed! Thanks @ahunter1 and welcome.

**Comment by MattHJensen** [comment]
Looks great. Merging. Thanks @ahunter1!

**Pull request merged**